apisix/plugins/limit-count.lua

```diff
@@ -192,7 +248,12 @@ function _M.access(conf, ctx)
         key = ctx.var["remote_addr"]
     end
 
-    key = key .. ctx.conf_type .. ctx.conf_version
+    if not conf.group then
+        key = key .. ctx.conf_type .. ctx.conf_version
+    else
+        key = key .. conf.group
```

Comment:
So traffic will eventually be limited according to the key generated by the combination of key and group.
But `key` can not be empty (if `key` is empty, `remote_addr` will be set as the default key), 
so it is never possible to limit the rate only according to the `group`.

Reply:
Yes, the previous limit-count is two-dimension:
per-route x per-key

This PR allows to use:
group x per-key

If we want to limit only according to the group, we can use a constant variable, like hostname, so the per-key will be always a single point.

In the next PR, I will introduce a new key type that is equal to a constant variable, so we don't need a special configuration for the key part.

Reply:
Look like the hostname is not constant if the cluster limit-count is used. Anyway, we can solve it in the next PR.